.eslintrc.js

```diff
@@ -47,7 +51,36 @@ module.exports = {
       plugins: ['node'],
       rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
         // add your custom rules and overrides for node files here
+        "@typescript-eslint/explicit-function-return-type": ["off"],
+        "@typescript-eslint/no-var-requires": ["off"],
+        "no-dupe-else-if": ["off"],
+        "no-import-assign": ["off"],
+        "no-setter-return": ["off"],
+        "@typescript-eslint/camelcase": ["off"]
       })
-    }
+    },
+    {
+      files: ["*.ts"],
```

Comment:
Seems like this should be 

```suggestion
      files: ["**/*.ts"],
```